fhir-parent/pom.xml

```diff
@@ -276,23 +280,21 @@
                 <scope>test</scope>
             </dependency>
             <dependency>
-                <!--  Used by COS in fhir-bucket and bulkdata -->
-                <groupId>com.fasterxml.jackson.core</groupId>
-                <artifactId>jackson-databind</artifactId>
-                <version>2.12.3</version>
```

Comment:
will this prevent us from controlling the version of jackson-databind used by COS?

Reply:
great catch... I'll fix this

Reply:
fixed, waiting on the CI

Reply:
I see you added jackson-databind back, but I'm thiinking maybe it should be using this new `${jackson.version}` variable.  Thoughts?
BTW, this is such a big PR that I'm totally fine to merge as-is and tweak the pom dependencies in a standalone one.

Reply:
so jackson gets kind of tricky as they version independently, leaving as is is OK, and case-by-case basis we can change it

Reply:
aka - I think that is a standalone activity